controllers/toolchaincluster/toolchaincluster_controller_test.go

```diff
@@ -94,35 +93,80 @@ func TestClusterControllerChecks(t *testing.T) {
 
 		cl := test.NewFakeClient(t, stable, sec)
 		reset := setupCachedClusters(t, cl, stable)
+
 		defer reset()
 		controller, req := prepareReconcile(stable, cl, requeAfter)
 
 		// when
-		recresult, err := controller.Reconcile(context.TODO(), req)
+		recResult, err := controller.Reconcile(context.TODO(), req)
 
 		// then
 		require.Equal(t, err, nil)
-		require.Equal(t, reconcile.Result{RequeueAfter: requeAfter}, recresult)
-		assertClusterStatus(t, cl, "stable", healthy())
+		require.Equal(t, reconcile.Result{RequeueAfter: requeAfter}, recResult)
+		assertClusterStatus(t, cl, "stable", clusterReadyCondition())
 	})
 
 	t.Run("toolchain cluster cache not found", func(t *testing.T) {
+		// given
+		unstable, _ := newToolchainCluster("unstable", tcNs, "http://unstable.com", toolchainv1alpha1.ToolchainClusterStatus{})
+
+		cl := test.NewFakeClient(t, unstable)
+		controller, req := prepareReconcile(unstable, cl, requeAfter)
+
+		// when
+		_, err := controller.Reconcile(context.TODO(), req)
+
+		// then
+		require.EqualError(t, err, "cluster unstable not found in cache")
+		actualToolchainCluster := &toolchainv1alpha1.ToolchainCluster{}
+		err = cl.Client.Get(context.TODO(), types.NamespacedName{Name: "unstable", Namespace: tcNs}, actualToolchainCluster)
+		require.NoError(t, err)
+		assertClusterStatus(t, cl, "unstable", clusterOfflineCondition("cluster unstable not found in cache"))
```

Comment:
there is no need to call explicitly, when it's called inside of the `assertClusterStatus` function:
```suggestion
		assertClusterStatus(t, cl, "unstable", clusterOfflineCondition("cluster unstable not found in cache"))
```